docs/production-deployment/cloud/introduction/security.mdx

```diff
@@ -159,7 +161,7 @@ Once set up, you can test your PrivateLink connectivity using:
       temporal.api.workflowservice.v1.WorkflowService/GetSystemInfo
   ```
 
-- Temporal SDKs: by setting the endpoint server address argument to the PrivateLink endpoint (`<DNS associated with VPC endpoint>:7233`).
+- Temporal SDKs, by setting the endpoint server address argument to the PrivateLink endpoint (`<DNS associated with VPC endpoint>:7233`) and using the TLS configuration options to override the TLS server name.
```

Comment:
This reads weird to me without the comma or colon

Reply:
Comma it is